index.html

```diff
@@ -12,42 +12,34 @@
   <title>John Mutisya's Portofolio</title>
 </head>
 <body>
-
 <section class="main-wrapper">
   <header>
     <nav class="navbar">
       <a href="#" class="nav-left">John Mutisya</a>
-
       <ul class="menu-list">
-
-      <li class="menu-list-items"><a class="menu-list-link" href="#">Portofolio</a></li>
-      <li class="menu-list-items"><a class="menu-list-link" href="#">About</a></li>
-      <li class="menu-list-items"><a class="menu-list-link" href="#">Contact</a></li>
-       
+        <li class="menu-list-items"><a class="menu-list-link" href="#Portofolio">Portofolio</a><span class="mobile-span"> > </span></li>
+        <li class="menu-list-items"><a class="menu-list-link" href="#About">About</a> <span class="mobile-span"> > </span> </li>
+        <li class="menu-list-items"><a class="menu-list-link" href="#Contact">Contact</a> <span class="mobile-span"> ></span> </li>
         <li class="menu-list-items"><img class="menu-list-link mail-icon" src="./images/mail-icon-removebg-preview.png" alt=""></li>
```

Comment:
[OPTIONAL]

- I can see a white space by the side of your mobile view which I would recommend you remove as it is not making the webpage look good and user-friendly. As you move from desktop to mobile view, the original background should be maintained without adding white space.

![image](https://user-images.githubusercontent.com/63473584/188722385-840b7558-589f-4475-b194-90789bacdb06.png)